README.md

```diff
@@ -1,6 +1,7 @@
 Fission: Serverless Functions for Kubernetes
 ============================================
 [fission.io](http://fission.io)  [@fissionio](http://twitter.com/fissionio)
+[![Build Status](https://travis-ci.org/platform9/fission.svg?branch=master)](https://travis-ci.org/platform9/fission)
```

Comment:
Hmm, we expect a lot of the tests to fail right now: controller tests assume a running etcd on localhost; poolmgr will require a running kubernetes cluster to test.  I'd suggest we hold off on enabling the badge until we get some passing tests.